src/meta/play-meta.js

```diff
@@ -243,6 +244,17 @@ export const plays = [
     language: 'js',
     featured: true,
   }, {
+    id: 'pl-tic-tac-toe-game', 
+    name: 'Tic Tac Toe Game', 
+    description: 'This game is coded in ReactJS and VanillaCSS',
+    component: () => {return <TicTacToeGame />},
+    path: '/plays/tic-tac-toe-game',
+    level: 'Beginner',
+    tags: 'Hooks,JSX,Functions,ResetState,CSS',
+    github: 'tejinder-sharma',
+    cover: '',
+    blog: '',
+    video: '',
```

Comment:
The merge conflict was not resolved correctly. It is overlapped with the meta data of another play. Please take care.

Reply:
I have resolved the CSS conflict but am not able to resolve the merge conflict.

I am not able to resolve merge conflicts, please suggest how to do it?

Reply:
I am getting errors while running the play locally. The errors are related to calendar play. please suggest how to resolve this issue.
<img width="1420" alt="Screenshot 2022-05-23 at 7 56 14 PM" src="https://user-images.githubusercontent.com/95734921/169842468-d5ae92cf-9548-4c51-b6cc-8d66aa3ff7e8.png">

Reply:
Please do a yarn or npm install again.